spec/actions_specs/changelog_from_git_commits_spec.rb

```diff
@@ -49,7 +49,7 @@
         end.to raise_error(":between must be of type array".red)
       end
 
-      it "Does not accept an array of size 1" do
+      it "Does not accept a :between array of size 1" do
```

Comment:
Yep. When I added the new test I felt like those messages could use more clarification about where this "array" we were testing was coming from.